src/jinja2/compiler.py

```diff
@@ -1617,13 +1633,6 @@ def visit_NSRef(self, node: nodes.NSRef, frame: Frame) -> None:
         # `foo.bar` notation they will be parsed as a normal attribute access
         # when used anywhere but in a `set` context
```

Comment:
Is this comment still relevant with all this removed? Is this visitor method still relevant?